### PR TITLE
Fix #32: Status bar when focus is lost

### LINKED ISSUE #32: Background Refresh Time Incorrect When Inactive
When the application is inactive the background refresh stops.
However, the toolbar at the bottom of the app still states a refresh is in place.

### PR DESCRIPTION
**Pull request by object71**
## What Changed?

Changed the status bar to be set on lose focus so that it can show the relevant information.

## Issues Closed
Fixes #32

### CODE REVIEW

File: src/ServiceBouncer/MainForm.cs [hunk 1]
```diff
@@ -321,6 +324,19 @@ private void SetTitle()
...
+        private void SetConnectedStatusBar()
+        {
+            if (isActive)
+            {
+                var backgroundRefreshTimeText = backgroundRefreshSeconds == 1 ? "1 second" : $"{backgroundRefreshSeconds} seconds";
+                toolStripStatusLabel.Text = $@"Connected to {machineHostname}. - Background refresh every {backgroundRefreshTimeText}.";
+            }
+            else
+            {
+                toolStripStatusLabel.Text = $@"Connected to {machineHostname}.";
```
Comment (BlythMeister) [inline src/ServiceBouncer/MainForm.cs]: Could this message state no refresh please.

**Review by BlythMeister — CHANGES_REQUESTED** [review]
One tiny change requested

File: src/ServiceBouncer/MainForm.cs (hunk 1 above)
Comment (object71) [reply src/ServiceBouncer/MainForm.cs]: Remove the refresh status message completely?

Comment (BlythMeister) [reply src/ServiceBouncer/MainForm.cs]: Sorry, no, that wasn't a clear comments.
Can you set the text to "Connected..... Background Refresh Disabled"

Comment (object71) [reply src/ServiceBouncer/MainForm.cs]: Yes, of course :)

**Comment by object71** [comment]
Change made to state "- Background refresh disabled"

**Comment by object71** [comment]
closed accidentally

**Review by BlythMeister — APPROVED** [review]
Great thanks

**Pull request merged**